saleor/graphql/webhook/types.py

```diff
@@ -22,6 +27,56 @@ def resolve_name(root: models.WebhookEvent, *_args, **_kwargs):
         return WebhookEventType.DISPLAY_LABELS.get(root.event_type) or root.event_type
 
 
+class EventDeliveryAttempt(CountableDjangoObjectType):
+    created_at = graphene.DateTime(required=True)
+    status = EventDeliveryStatusEnum(
+        description="Event delivery status.", required=True
+    )
+
+    class Meta:
+        description = "Webhook delivery attempts."
+        model = core_models.EventDeliveryAttempt
+        interfaces = [graphene.relay.Node]
+        only_fields = [
+            "id",
+            "created_at",
+            "task_id",
+            "duration",
+            "status",
+            "response",
+            "response_headers",
+            "request_headers",
+        ]
+
+
+class EventDelivery(CountableDjangoObjectType):
+    status = EventDeliveryStatusEnum(
+        description="Event delivery status.", required=True
+    )
+    attempts = FilterInputConnectionField(
+        EventDeliveryAttempt,
+        sort_by=EventDeliverySortingInput(description="Event delivery sorter"),
+        description="Webhook delivery attempts.",
+    )
+    event_type = WebhookEventTypeEnum(description="Webhook event type.", required=True)
+
+    class Meta:
+        description = "Event delivery."
+        model = core_models.EventDelivery
+        interfaces = [graphene.relay.Node]
+        only_fields = [
+            "id",
+            "created_at",
+            "status",
+            "event_type",
+        ]
+
+    @staticmethod
+    def resolve_attempts(root: core_models.EventDelivery, info, *_args, **_kwargs):
+        AttemptsByDeliveryLoader(info.context).load(root)
```

Comment:
The dataloader is called here but its results are not returned from the resolver.

Reply:
As discussed  - we will leave FilterInputConnectionField fields without data loaders for now